Apromore-Calendar/src/main/java/org/apromore/calendar/model/CalendarModel.java

```diff
@@ -32,6 +32,7 @@
 import java.time.temporal.ChronoUnit;
 import java.util.ArrayList;
 import java.util.Collections;
+import java.util.Comparator;
 import java.util.List;
 import java.util.Map;
 import java.util.function.Function;
```

Comment:
Please add some comments on the key model classes regarding what they represent and how they should be used (e.g. what is the entry point, their relationships, etc.), or point to design document (if any). These classes need header comments: CalendarModel, WorkDayModel, HolidayModel, CalendarService). These brief comments would be helpful for users of this service to understand and use them properly (of course you have explained them in meetings, but we need to read these comments at times).

Reply:
Will Do